doc/filters/merge.rst

```diff
@@ -1,7 +1,9 @@
 ``merge``
 =========
 
-The ``merge`` filter merges an array with another array:
+The ``merge`` filter merges sequences and mappings.
+
+The ``merge`` filter also works on sequences:
```

Comment:
The "also" here and below reads weird. What about merging both code blocks instead?

Reply:
I've done it a bit differently, it's better now.